backend/app/controllers/application_controller.rb

```diff
@@ -1,4 +1,15 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::API
+  def authorize_request
+    header = request.headers["Authorization"]
+    token = header.split(" ").last if header
+
+    begin
+      decoded = JwtService.decode(token)
+      @user = User.find(decoded[:user_id])
+    rescue
```

Comment:
I do not recommend `blanket rescue` statements because it might rescue errors that you do not expect, but point to another problem in the code.

Please specify the error you are rescuing. What is the JWT error if the token is invalid? Or if User.find is invalid?